src/components/Form/Collection/Collection.jsx

```diff
@@ -242,6 +242,22 @@ export default class Collection extends ValidationElement {
       )
     }
 
+    // Create the 'byline' which is used to separate items as well as the bottom toggle action
+    const bylineBottom = (item, index) => {
+      if (this.state.items.length < 2) {
+        return ''
+      }
+
+      return (
+        <div className="byline bottom">
+          <a href="javascript:;;" className="toggle" onClick={this.toggle.bind(this, index)}>
```

Comment:
I personally use to (and still do depending on the product) use

```html
<a href="#">...</a>
```

However using hashtag routing in single page applications doesn't always play nice with that. :smile: